content/en/docs/Kyverno CLI/_index.md

```diff
@@ -224,3 +224,159 @@ policies:
 ```
 kyverno apply /path/to/add_network_policy.yaml --resource /path/to/required_default_network_policy.yaml -f /path/to/value.yaml
 ```
+
+#### Policy Report
+
+Use '--policy_report' with apply command to generate policy report.
+
+Input combination:
+
+| S.No | policy        | resource         | cluster   | namespace      | interpretation                                                                           |
```

Comment:
What does "S.No" mean? Column names should be initially capitalized.

Reply:
"S.No" - just rows count